CodeEdit/Features/CEWorkspace/Models/DirectoryEventStream.swift

```diff
@@ -111,7 +111,7 @@ class DirectoryEventStream {
         _ eventFlags: UnsafePointer<FSEventStreamEventFlags>,
         _ eventIds: UnsafePointer<FSEventStreamEventId>
     ) {
-        var eventPaths = eventPaths.bindMemory(to: UnsafePointer<CChar>.self, capacity: numEvents)
+        let eventPaths = eventPaths.bindMemory(to: UnsafePointer<CChar>.self, capacity: numEvents)
```

Comment:
This was to fix a swift lint error, because `eventPaths` is not mutated at all and should be a `let`.